va/va.h

```diff
@@ -495,6 +542,30 @@ typedef enum
      */
     VAConfigAttribEncRateControlExt   = 26,
 
+    /**
+     * \brief Encode function type for FEI.
+     *
+     * This attribute conveys whether the driver supports different function types for encode.
+     * It can be ENC, PAK, or ENC + PAK. Currently it is for FEI entry point only.
+     * Default is ENC + PAK.
```

Comment:
Would better to change the ENC/PAK/ENC+PAK to be  VA_FEI_FUNCTION_ENC， VA_FEI_FUNCTION_PAK and VA_FEI_FUNCTION_ENC_PAK. It should to be clearer.